pkg/daemon/update.go

```diff
@@ -2127,6 +2127,13 @@ func (dn *Daemon) experimentalUpdateLayeredConfig() error {
 		return nil
 	}
 
+	desiredConfig := dn.node.Annotations[constants.DesiredMachineConfigAnnotationKey]
```

Comment:
Yep, awesome, you still call both`SetDone()` and `SetLayeredDone()` for now so it should still be happy regardless.